src/test/java/org/opentripplanner/middleware/TestUtils.java

```diff
@@ -47,6 +48,12 @@ public static boolean getBooleanEnvVar(String var) {
         return variable != null && variable.equals("true");
     }
 
+    /**
+     * Helper method to determine if end to end is enabled and auth is disabled. (Used for checking if tests should run.)
+     */
+    public static boolean isEndToEndAndAuthIsDisabled() {
+        return getBooleanEnvVar("RUN_E2E") && authDisabled();
```

Comment:
Philosophical question: do we still need to have the DISABLE_AUTH config parameter in that case?